File: app/src/main/java/com/iamkurtgoz/samplepreference/SampleApp.java
package com.iamkurtgoz.samplepreference;

import android.app.Application;

import com.iamkurtgoz.easystore.EasyStoreMode;
import com.iamkurtgoz.easystore.EasyStore;

public class SampleApp extends Application {

    @Override
    public void onCreate() {
        super.onCreate();

        EasyStore.getInstance().init(getApplicationContext(), "mypreference", EasyStoreMode.MODE_PRIVATE, true, "ASDASDASD");
    }
}

File: app/src/main/java/com/iamkurtgoz/samplepreference/SampleApp.kt
package com.iamkurtgoz.samplepreference

import android.app.Application
import com.iamkurtgoz.easystore.EasyStore
import com.iamkurtgoz.easystore.EasyStoreBuilder
import com.iamkurtgoz.easystore.EasyStoreMode

class SampleApp : Application() {

    override fun onCreate() {
        super.onCreate()
        //EasyStore.getInstance().init(this, "defaultName", EasyStoreMode.MODE_PRIVATE)

        EasyStoreBuilder(this)
    }
}